components/Chat/Input.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { StyleSheet, View, ViewStyle, TextInput } from 'react-native';
+import { AppTheme } from '../../config/DefaultConfig';
+import useTheme from "../../hooks/useTheme";
+import { isIOS } from '../../utils';
+import ErrorText from './ErrorText';
+
+interface Props {
+    placeholder: string;
+    value?: string;
+    onChange?: any;
+    secureCheck?: any;
+    validation?: any;
+};
```

Comment:
This is not how it should be we should have extended the `TextInputProps` to add new props

Reply:
I have extended the TypeInputProps